processors/gtfs.py

```diff
@@ -309,66 +291,75 @@ def add_stop_gtfs(route_stop, city):
-                        dict_to_row(
-                            {
-                                "trip_id": variant.id,
-                                "stop_sequence": stop_sequence,
-                                "shape_dist_traveled": route_stop.distance,
-                                "stop_id": gtfs_platform_id,
-                            },
-                            "stop_times",
-                        )
+                        {
+                            "trip_id": variant.id,
+                            "stop_sequence": stop_sequence,
+                            "shape_dist_traveled": route_stop.distance,
+                            "stop_id": gtfs_platform_id,
+                        }
                     )
 
     # stops
-    gtfs_data["stops"].extend(
-        map(lambda row: dict_to_row(row, "stops"), all_stops.values())
-    )
+    gtfs_data["stops"].extend(all_stops.values())
 
     # transfers
     for stoparea_set in transfers:
         for stoparea1 in stoparea_set:
             for stoparea2 in stoparea_set:
                 if stoparea1.id < stoparea2.id:
+                    stop1_id = f"{stoparea1.id}_st"
+                    stop2_id = f"{stoparea2.id}_st"
+                    if not all(
+                        stop_id in all_stops
+                        for stop_id in (stop1_id, stop2_id)
+                    ):
+                        continue
                     transfer_time = TRANSFER_PENALTY + round(
                         distance(stoparea1.center, stoparea2.center)
                         / SPEED_ON_TRANSFER
                     )
                     for id1, id2 in (
-                        (stoparea1.id, stoparea2.id),
-                        (stoparea2.id, stoparea1.id),
+                        (stop1_id, stop2_id),
+                        (stop2_id, stop1_id),
                     ):
                         gtfs_data["transfers"].append(
-                            dict_to_row(
-                                {
-                                    "from_stop_id": f"{id1}_st",
-                                    "to_stop_id": f"{id2}_st",
-                                    "transfer_type": 0,
-                                    "min_transfer_time": transfer_time,
-                                },
-                                "transfers",
-                            )
+                            {
+                                "from_stop_id": id1,
+                                "to_stop_id": id2,
+                                "transfer_type": 0,
+                                "min_transfer_time": transfer_time,
+                            }
                         )
 
     make_gtfs(filename, gtfs_data)
 
 
+def dict_to_row(dict_data, record_type):
+    """Given object stored in a dict and an array of columns,
+    returns a row to use in CSV.
+    """
+    row = []
+    for column in GTFS_COLUMNS[record_type]:
+        value = dict_data.get(column)
```

Comment:
Instead of 
```python
        if value is None:
            value = ""
```
You can set default value for `get`:
```suggestion
        value = dict_data.get(column, "")
```

Reply:
\+